### ISSUE TITLE
Links have insufficient color contrast ratio

### ISSUE THREAD

**Issue by rpsychologist** [biostatistics-psychiatry/lab-materials#57]
The links (#18bc9c on white background) only have a contrast ratio of 2.41 which does not meet accesability requirements.
